lib/avo/fields/key_value_field.rb

```diff
@@ -32,6 +28,30 @@ def initialize(id, **args, &block)
         @disable_deleting_rows = args[:disable_deleting_rows].present? ? args[:disable_deleting_rows] : false
       end
 
+      def key_label
+        return @key_label if @key_label && !@key_label.empty?
```

Comment:
Can we use `.present?` here and below?

```suggestion
        return @key_label if @key_label.present?
```